app/views/nic/bridges/_bridge.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render 'nic/base_form', :f => f %>
+
+<%= content_tag :span, :class => 'bridge_fields' do %>
+  <%= text_f f, :attached_devices, :help_inline => _('comma separated interface identifiers'), :class => 'attached' %>
```

Comment:
I think this has to be slightly modified, otherwise the form seems a bit broken with current develop. Here's the diff that fixed it for me.

```diff
diff --git a/app/views/nic/bridges/_bridge.html.erb b/app/views/nic/bridges/_bridge.html.erb
index ab685a5..b4c8d92 100644
--- a/app/views/nic/bridges/_bridge.html.erb
+++ b/app/views/nic/bridges/_bridge.html.erb
@@ -1,7 +1,7 @@
 <%= render 'nic/base_form', :f => f %>
 
 <%= content_tag :span, :class => 'bridge_fields' do %>
-  <%= text_f f, :attached_devices, :help_inline => _('comma separated interface identifiers'), :class => 'attached' %>
+  <%= text_f f, :attached_devices, :help_inline => popover('', _('comma separated interface identifiers'), :rel => 'popover-modal'), :class => 'attached', :size => "col-md-8", :label_size => "col-md-3" %>
 <% end %>
 
 <%= render 'nic/provider_specific_form', :f => f %>
```
![Uploading bridge.png…]()